sdks/urbackend-python/test_sdk.py

```
@@ -0,0 +1,371 @@
+# Ensure we import the local SDK, not any installed version
+sys.path.insert(0, os.path.dirname(__file__))
+
+from urbackend import UrBackendClient, AuthError, NotFoundError, ValidationError
+
+# ────────────────────────────────────────────────────────────────
+#  CONFIG — Replace these with your actual values
+# ────────────────────────────────────────────────────────────────
+API_KEY    = "YOUR_PUBLISHABLE_KEY"        # ← your publishable key
+BASE_URL   = "YOUR_BASE_URL"    # ← or http://localhost:1235
+EMAIL      = "YOUR_EMAIL"
+PASSWORD   = "YOUR_PASSWORD"
```

Ruff (0.15.11)

[error] Possible hardcoded password assigned to: "PASSWORD"

(S105)